docs/source/user_guide/faq.rst

```diff
@@ -0,0 +1,118 @@
+    strongly recommend that you use a GPU. With that being said, an integrated GPU should be sufficient for many
+    use-cases.
+
+How does `fastplotlib` relate to `matplotlib`?
+----------------------------------------------
+
+    `fastplotlib` is **not** related to `matplotlib` in any way.
+
+    These are two completely different libraries with their own APIs and use-cases. The `fastplotlib` library is primarily for *interactive*
+    visualization that runs on the GPU using WGPU. The `fastplotlib` architecture is completely different from `matplotlib`. Using `fastplotlib`
+    is more akin to using `numpy`. See the "How can I learn to use `fastplotlib`?" section below.
+
+How can I learn to use `fastplotlib`?
+-------------------------------------
+
+    We want `fastplotlib` to be easy to learn and use. To get started with the library we recommend taking a look at our `guide <https://fastplotlib.readthedocs.io/en/latest/user_guide/guide.html>`_ and
+    `examples gallery <https://fastplotlib.readthedocs.io/en/latest/_gallery/index.html>`_.
+
+    In general, if you are familiar with numpy and array notation you will already have a intuitive understanding of interacting
+    with your data in `fastplotlib`. If you have any questions, please do not hesitate to post an issue or discussion forum post.
+
+Should I use `fastplotlib` for making publication figures?
+----------------------------------------------------------
+
+    No, `fastplotlib` is not meant for creating *static* publication figures. There are many other libraries that are well-suited
+    for this task.
+
+How does `fastplotlib` handle data loading?
+-------------------------------------------
+
+    `fastplotlib` is a plotting library and not a data handling or data loading library. These tasks are outside of the scope of
+    the library.
+
+    In general, if your data is an array-like object, `fastplotlib` should be able to use it. However, if you have any problems using your data objects,
+    please do not hesitate to post an issue!
+
+What is the scope of `fastplotlib`?
+-----------------------------------
+
+    While the capabilities are very far-reaching, we would like to emphasize that `fastplotlib` is a general-purpose plotting library focused on scientific visualization.
+    More specifically, we aim to develop the tools necessary for users to build fast and interactive visualizations for a variety of scientific domains (e.g. neuroscience,
+    astrophysics). If you have a particular feature in mind that you feel is missing, please post an issue and we will respond accordingly letting you know if it fits within
+    the scope of the project.
+
+What types of PRs are we willing to accept?
+-------------------------------------------
+
+    Primarily the features of `fastplotlib` have been developed as they relate to the core-developers research use cases (mostly neuroscience). With that being said, there are many domains in which
```

Comment:
add more to our research use case:

(neuroscience, algorithm development, and machine learning), it's more broad XD